File: 2020-Spring/COMSC-275 Intro to Web Programming/Assignments/14. Time/PHP/dateTime.php
 <?php

date_default_timezone_set('US/Pacific');

$dateTime=date('l F j, Y H:i:s a e');

header('Content-type: text/xml');

print "<?xml version=\"1.0\" ?>";

print "Curren Date and Time: <dateTime>$dateTime</dateTime>";

?>

File: 2020-Spring/COMSC-275 Intro to Web Programming/Assignments/14. Time/Python/dateTime.py
#! /Python27/Python


from datetime import datetime

dateTime = datetime.now()

print("Content-type: 'text/xml' \n\n")

print("<?xml version='1.0' ?>")

print("Current Date and Time: ")

print("<dateTime>")

print(dateTime)

print("</dateTime>")